Inception/Inception/Screens/AlarmList/Cell/AlarmListCell.swift

```diff
@@ -29,21 +29,14 @@ class AlarmListCell: UITableViewCell {
     sleepHourIcon.image = UIImage(systemName: "bed.double.fill", withConfiguration: configuration)
   }
   
-  override func setSelected(_ selected: Bool, animated: Bool) {
-    super.setSelected(selected, animated: animated)
+  func alarmCellUpdate(with alarm: Alarm, isSetAlarm: Bool) {
```

Comment:
```suggestion
  func alarmCellUpdate(with alarm: Alarm, isSet: Bool) {
```
p4;

매개변수 이름 추천드립니다~
`isSet` 또는 `isOn` 으로 작성해도 의미 전달이 잘 될 거 같아요!

Reply:
이 부분도 기존/현 작업자가 알로라라서 상의후에 수정결정하도록 하겠습니다 
@compuTasha  알로라 요부분도 제가 건드려도될까요?

Reply:
현재 `AlarmListViewController` 에서 저장된 알람 목록 cell에 opacity를 주고 있어서
setSelected를 포함한 isSet 부분을 삭제해도 될 것 같습니다..!!

Reply:
넵 변경했습니다 2886bea8c33a896b36140f32f8ae3b615943f3ea